tests/console/snapper_undochange.pm

```diff
@@ -15,22 +21,24 @@ use testapi;
 sub run() {
     select_console 'root-console';
 
-    my $snapfile = '/root/snapfile';
-
-    my $snapbf = script_output "snapper create -p -d 'before undochange test'";
-    script_run "date > $snapfile";
-    my $snapaf = script_output "snapper create -p -d 'after undochange test'";
+    my $snapfile = '/root/test_snapfile';
+    # snapper undochange [options] number1..number2 [files]
 
+    script_run("snapper create -p -d 'before undochange test'", 0);
+    my $snapbf = script_output("snapper list | grep before undochange | cut -d'|' -f2 | xargs");
+    script_run("date > $snapfile");
+    script_run("snapper create -p -d 'after undochange test'", 0);
+    my $snapaf = script_output("snapper list | grep after undochange | cut -d'|' -f2 | xargs");
     # Delete snapfile
-    script_run "snapper undochange $snapbf..$snapaf $snapfile";
+    script_run("snapper undochange $snapbf..$snapaf $snapfile",              0);
```

Comment:
i will not add this, but it for understanding why  script_run("snapper undochange $snapbf..$snapaf $snapfile",  0") work internally.